cmake/package_settings.cmake

```diff
@@ -39,30 +39,21 @@ write_basic_package_version_file(
 install(
   FILES ${CMAKE_BINARY_DIR}/cmake/openenclave-config.cmake
   ${CMAKE_BINARY_DIR}/cmake/openenclave-config-version.cmake
-  DESTINATION ${CMAKE_INSTALL_LIBDIR}/openenclave/cmake)
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/openenclave/cmake
+  COMPONENT OEHOSTVERIFY)
 install(
   EXPORT openenclave-targets
   NAMESPACE openenclave::
   # Note that this is used in `openenclaverc` to set the path for
   # users of the SDK and so must remain consistent.
   DESTINATION ${CMAKE_INSTALL_LIBDIR}/openenclave/cmake
-  FILE openenclave-targets.cmake)
+  FILE openenclave-targets.cmake
+  COMPONENT OEHOSTVERIFY)
```

Comment:
What scenario _won't_ these be required? (I'm trying to understand the choice of components.)

Reply:
It will always be required I think. To create the full SDK, you do a non-component CPack, which will include everything. To create the host verification library, you do a component install w/ component=OEHOSTVERIFY.